### ISSUE TITLE
Article: DPC Blog on Diversity, Let's Get Together and Feel Alright...

### ISSUE THREAD

**Issue by ross-spencer** [digital-preservation-is-people/digital-preservation-is-people.github.io#5]
http://www.dpconline.org/blog/diversity-blog

**Comment by andreakb** [comment]
This is interesting, a step in the right direction, I do wonder about this though: 
![image](https://user-images.githubusercontent.com/5824921/35575949-cb11287e-05ab-11e8-83d4-88e71b89172d.png)

This is something I see or hear when talking to people I consider to be reasonable, a reticence to call something a code of conduct, or bemoaning that it's a sad state that we have to have a code of conduct.

**Comment by kfrn** [comment]
Hmm, it seems like a dubious attempt to rebrand a CoC.

**Comment by andreakb** [comment]
Totally! I'm glad they're doing this work, but I'm wondering why commitments & codes are mutually exclusive? You can do this work, and also define what harassment is and how you're org deals with harassment ...    Thinking about this kind of in terms of say, bike helmets. There are places where bike helmets are definitely needed (NYC, Wellington for instance)! And places where you can bike safely without an helmet (Kytoto, Copenhagen I think?) , but those places that don't need helmets have so many systems, norms, and safety infrastructure in place to make a helmet unnecessary. The goal of course, would be that you could bike anywhere without a helmet, because it would be safe enough! And helmets still exist, will always exists, even in those safer cities! But it takes sooooo much work to make cities safe for biking and a lot of cities aren't up for the task/don't want to do the work/don't want to lose parking spaces and driving lanes for the more powerful cars.  I think there is an analogy there. Does that make sense?

**Comment by kfrn** [comment]
I love your bike helmet metaphor, @andreakb. (As an aside, it made me think of this tweet that popped up on my timeline yesterday: https://twitter.com/Ghetsuhm/status/957742232479477760)

I find it interesting that she presents 'Code of Conduct' as a negatively-coded phrase ("we don’t want negative interactions to be the main focus"). Maybe that's the common view (? - IDK), but for me it's quite a value-neutral phrase. It seems like a CoC is a place where you can lay out expectations for respectful behaviour, as well as consequences for harassment etc.

**Comment by SharonMcMeekin** [comment]
Hey Guys, thought I should just pop my head up and say hello. Ross added me to the group.

Perhaps my wording in the blog post isn't totally ideal, I 100% believe that the commitments laid out in CoCs are essential. However, discussions with the many and varied people we meet in the DPC have led us to understand that CoC is, unfortunately, a loaded-term for some, even if that isn't the intention behind the documents themselves. Having read quite a lot of them recently (from a number of different disciplines) I can kinda see why, many do place emphasis on building productive and inclusive environments, but others jump straight into harassment and don't really touch on anything else.

I **love** the bike helmet analogy, it works well for what we want to achieve with this, and why we've started to see this work as more than just a CoC. We're hoping to work to encourage that infrastructure to allow us to help ensure the welfare of all we interact with, especially in spaces managed with the DPC. It will, therefore, add explicit commitments from the DPC about how we will support issues around both physical and mental health. As a person who sometimes struggles with the intensity of in-person events (particularly conferences), this is an issue that is very close to my heart! We'll also be developing guidance for DPC staff on how to provide this support. I'm hoping to do some proper training myself in this area so these documents will reflect best practice. I'm planning to get advice on where best to source this from friends who work with Deafblind Scotland and LGBT Youth Scotland.

Its very early days for us looking at this topic and I don't know yet what the final product will look like, but I'd definitely welcome all thoughts and input you have!

**Comment by andreakb** [comment]
Thanks, Sharon,for joining, saying hello and for being so thoughtful in your explanation! When I hear some folks talking about how a CoC is a loaded term it's kind of a whiplash, wait whaaaaat, moment for me, because as Katherine says, to me it's also pretty value-neutral phrase. I understand that other people have different ... understandings, I guess? 

I'm so so so glad that you and the other folk at DPC are taking on this work, and taking it seriously! I was really excited to see this post, and I'm looking forward to see what the DPC comes up with. You're absolutely right, it's more than a CoC. Sometimes a CoC can be a bare minimum or not even effective if you don't have the proper infrastructure or training to support it. I think, if you're hesitant to call something a CoC, a good example to look at would be Allied Media Projects Conference. They have  [social media and participant agreements](https://www.alliedmedia.org/amc/agreements) along with  [participant rights and accessibility statements](https://www.alliedmedia.org/amc/accessibility). Ashe Drydan also has some good resources, if you want to look her up. She has a [code of conduct 101](https://www.ashedryden.com/blog/codes-of-conduct-101-faq) that may be useful in the early days, and addresses some of those questions about CoC s seeming punitive, etc.

**Comment by SharonMcMeekin** [comment]
Yeah, personally I find the existence of CoCs to be a positive and reassuring thing. Especially when you know they are backed up with both proper consideration and procedures. Thanks so much for suggesting those resources, I'll put them at the top of my 'to read' list!
